autoconf/m4/nl_enable_coverage.m4

```diff
@@ -53,7 +54,7 @@ AC_DEFUN([NL_ENABLE_COVERAGE],
         [m4_fatal([$0: invalid default value '$1'; must be 'yes' or 'no'])])
 
     AC_CACHE_CHECK([whether to build code-coverage instances of programs and libraries],
-        nl_cv_build_coverage,
+        [nl_cv_build_coverage],
```

Comment:
what is the significance of the square brackets here?

Reply:
In this case, it should be a no-op.

In investigating the issue yesterday and going back to the canonical AC_CACHE_CHECK examples in the online documentation, it always showed these values as quoted (where '[' and ']' are quotes in M4). I added the quotes during my experiments but didn't pull them back out.